ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -492,13 +529,25 @@ class MesheryAdapterPlayComponent extends React.Component {
 
   render() {
     const {
-      classes, adapter, adapter_icon,
+      classes, adapter,
     } = this.props;
     const {
       namespace,
       namespaceError,
     } = this.state;
 
+    let adapterName = (adapter.name).split(" ").join("").toLowerCase();
+    let imageSrc = "/static/img/" + adapterName + ".svg";
+    let chp = (
```

Comment:
Can we rename it to adapterChip ?

Reply:
I can, but won't the adapterName be more effective, considering the data it contains, rather than adapterChip which doesn't quite goes with it. But if the other one seems better then, I will make a change.

Reply:
I was talking about ~chp~ -> adapterChip

Reply:
Ohh ok. Cool! 
Will change that :)

Reply:
Yes please!